nearform/react-pwa
# A11y

https://github.com/nearform/react-pwa/issues/81

Implements:
- skip links
- refactors some titles to use H1, H2 etc to create better document outline
- refactors navigation hamburger, close buttons, previos / next buttons to be actual buttons with proper labels, so we get a keyboard focus on them
- properly implements the side navigation (aria-hidden on main content when it's visible, focus trap on the navigation so you cant tab out of the navigation)

How to test:
- When the app loads, try pressing TAB key and observe skip links to be shown and the focus changing naturally e.g. not selecting hidden navigation elements like before.
- Try also mobile viewport and going to the navigation by keyboard.
- If you have `webdeveloper` toolbar extension installed in chrome, try viewint the document outline from the Information panel
